nav2_planner/src/planner_server.cpp

```diff
@@ -232,8 +232,11 @@ PlannerServer::computePlan()
       r.sleep();
     }
 
+    // Use start_pose if provided otherwise use current robot pose
     geometry_msgs::msg::PoseStamped start;
-    if (!costmap_ros_->getRobotPose(start)) {
+    if (goal->use_start_pose) {
+      start = goal->start;
```

Comment:
https://github.com/ros-planning/navigation2/issues/2186, it looks like we overlooked the stamped portion of the message for the goal (and now start). We should also check the `frame_id` of `start` and transform / print error if different from the global frame of the server's costmap used to plan in.

Reply:
The `costmap_ros_->getRobotPose(start)` does this implicitly